### ISSUE TITLE
crash on redirection?

### ISSUE THREAD

**Issue by dwery** [KraigM/homebridge-nest#4]
just got this:
```
undefined:1
<html>redirect to <a
href="https://czfe08-front01-iad01.transport.home.nest.co
^
SyntaxError: Unexpected token <
    at Object.parse (native)
    at IncomingMessage.<anonymous>
(/usr/lib/node_modules/homebridge-legacy-plugins/node_modules/unofficial-nest-api/index.js:167:40)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)
```

**Comment by KraigM** [comment]
That is unfortunately in the third party api.  Does it happen consistently or just the once?

**Comment by dwery** [comment]
Seen a couple of times. I had to disable the plugin in order to keep homebridge working.

**Comment by KraigM** [comment]
Setting up the new API on #11 should solve this issue (or rather avoid it entirely lol)

**Comment by KraigM** [comment]
The new API has been officially released to npm (as version 1.0.0).  For more info, see the [main page](https://github.com/KraigM/homebridge-nest/).  It should fix this issue.
